cmd/util/cmd/epochs/transactions/deploy_epoch_relative.cdc

```diff
@@ -0,0 +1,31 @@
+import FlowClusterQC from 0xQCADDRESS
+
+transaction(name: String, 
+            code: [UInt8],
+            currentEpochCounter: UInt64,
+            numViewsInEpoch: UInt64,
+            numViewsInStakingAuction: UInt64,
+            numViewsInDKGPhase: UInt64,
+            numCollectorClusters: UInt16,
+            FLOWsupplyIncreasePercentage: UFix64,
+            randomSource: String,
+            collectorClusters: [FlowClusterQC.Cluster]) {
+
+  prepare(signer: AuthAccount) {
+
+    let currentBlock = getCurrentBlock()
+
+    signer.contracts.add(name: name, 
+            code: code,
+            currentEpochCounter: currentEpochCounter,
+            numViewsInEpoch: numViewsInEpoch - currentBlock.view,
```

Comment:
This part _is_ assuming that the first view of the epoch is 0, should add that note here.